### ISSUE TITLE
"Help and Privacy" contains no Help.

### ISSUE THREAD

**Issue by chrisjj** [wasdk/WebAssemblyStudio#155]
Just a privacy statement:

![image](https://user-images.githubusercontent.com/6157073/36938754-322243c8-1f1e-11e8-8498-a0e0cd278be9.png)

**Comment by mbebenita** [comment]
Indeed, we haven't had a chance to write help yet. We're hoping to produce several introductory YouTube videos.

**Comment by haxxorsid** [comment]
Related: #75
